packages/create-cloudflare/templates/astro/c3.ts

```diff
@@ -46,9 +46,9 @@ const updateAstroConfig = () => {
 			n.node.arguments = [
 				b.objectExpression([
 					b.objectProperty(
-						b.identifier("runtime"),
+						b.identifier("platformProxy"),
 						b.objectExpression([
-							b.objectProperty(b.identifier("mode"), b.stringLiteral("local")),
+							b.objectProperty(b.identifier("enable"), b.booleanLiteral(true)),
```

Comment:
```suggestion
							b.objectProperty(b.identifier("enabled"), b.booleanLiteral(true)),
```

This should be `enabled` and not `enable` (https://github.com/withastro/adapters/pull/159/files#diff-1d2a2b56ad20e7e5b13936929d10bebf30d56ffa7cdc64f686dff4ae6cfafb60R52)

Reply:
🤦 thanks! 🙂